fili-core/src/main/java/com/yahoo/bard/webservice/druid/model/dimension/extractionfunction/TagExtractionFunctionFactory.java

```diff
@@ -0,0 +1,88 @@
+// Copyright 2019 Verizon Media Group
```

Comment:
My understanding was that this was not the correct copyright to use yet for our code. You may want to check this with Gil before committing it.